src/operator/src/error.rs

```diff
@@ -534,6 +534,13 @@ pub enum Error {
         source: session::session_config::Error,
         location: Location,
     },
+
+    #[snafu(display("Invalid timestamp range, start: `{}`, end: `{}`", start, end))]
```

Comment:
non-blocking: Maybe we could invoke timestamp's methods here.
```suggestion
    #[snafu(display("Invalid timestamp range, start: `{}`, end: `{}`", start.to_iso8601_string(), end.to_iso8601_string()))]
```

Reply:
I suppose it's already covered by:

```rust
            error::InvalidTimestampRangeSnafu {
                start: start.to_iso8601_string(),
                end: end.to_iso8601_string(),
            }
```